### PR TITLE
Add documentations for configurable wasm stack and heap sizes

### PR DESCRIPTION
**Pull request by cosmo0920**

### CODE REVIEW

File: pipeline/filters/wasm.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ The plugin supports the following configuration parameters:
 | Event\_Format | Define event format to interact with Wasm programs: msgpack or json. Default: json |
 | Function\_Name | Wasm function name that will be triggered to do filtering. It's assumed that the function is built inside the Wasm program specified above. |
 | Accessible\_Paths | Specify the whilelist of paths to be able to access paths from WASM programs. |
+| Wasm\_Heap\_Size | Size of the heap size of Wasm execution \(check [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values\) |
+| Wasm\_Stack\_Size | Size of the stack size of Wasm execution \(check [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values\) |
```
Comment (cnorris-cs) [inline pipeline/filters/wasm.md:26]:
```suggestion
| Wasm\_Heap\_Size | Size of the heap size of Wasm execution. Review [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values. |
| Wasm\_Stack\_Size | Size of the stack size of Wasm execution. Review [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values. |
```

File: pipeline/inputs/exec-wasi.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ The plugin supports the following configuration parameters:
 | Accessible\_Paths | Specify the whilelist of paths to be able to access paths from WASM programs. |
 | Interval\_Sec | Polling interval \(seconds\). |
 | Interval\_NSec | Polling interval \(nanosecond\). |
+| Wasm\_Heap\_Size | Size of the heap size of Wasm execution \(check [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values\) |
+| Wasm\_Stack\_Size | Size of the stack size of Wasm execution \(check [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values\) |
```
Comment (cnorris-cs) [inline pipeline/inputs/exec-wasi.md:17]:
```suggestion
| Wasm\_Heap\_Size | Size of the heap size of Wasm execution. Review [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values. |
| Wasm\_Stack\_Size | Size of the stack size of Wasm execution. Review [unit sizes](https://docs.fluentbit.io/manual/configuration/unit_sizes) for allowed values. |
```

**Review by cnorris-cs — APPROVED** [review]
I left suggestions, but I've added my approval based on that to keep this moving.

**Comment by patrick-stephens** [comment]
In release so merging

**Pull request merged**